src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The `draw()` method should not include the logic for building and printing the string representation of the object. Consider separating the concerns by creating a `toString()` method that returns the string representation, and then call it within the `draw()` method.